src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1184,14 +1184,33 @@ Prevent Long URL From Breaking Out of Container
     margin-right: 5px;
 }
 
-/*
-Glyphicon for success status messages
-*/
+/*//////////////////////////////////////////////////////////
+////////////////////// Status Icons //////////////////////
+//////////////////////////////////////////////////////////*/
 
-.icon-success:before {
+.icon-success:before,
+.icon-warning:before,
+.icon-danger:before,
+.icon-info:before {
+    /*noinspection CssNoGenericFontName*/
```

Comment:
I agree with @whipermr5. Since we don't recommend only a single IDE, our codebase should only include lines that are to do with our Application and nothing else. Plus, I don't know about IntelliJ, but at least in Eclipse, as of now, there are a lot of warnings too. If we were to identify and suppress each of them our code would not only become messy but also unreadable. This argument is compounded by the fact that IDE warnings are generally easy to ignore. I don't think this like should get special treatment.